de.dlr.sc.virsat.javadoc.api/help/api_docs.xml

```diff
@@ -2,265 +2,8 @@
 <?NLS TYPE="org.eclipse.help.toc"?><toc label="VirSat Core API JavaDoc Compilation">
 <topic href="help/api-docs/javadoc/overview-summary.html" label="VirSat Core API JavaDoc Compilation">
 <topic label="Packages">
-<topic href="help/api-docs/javadoc/com/github/cliftonlabs/json_simple/package-summary.html" label="com.github.cliftonlabs.json_simple"/>

```

Comment:
Suspicious changes

Reply:
Not an issue, the api_docs.xml gets regenerated by the maven build process when creating the java docs